### PR TITLE
Correctly detect Transmeta Crusoe as i686

### PR DESCRIPTION
**Pull request by simotek**
https://bugzilla.suse.com/show_bug.cgi?id=52713 (private bug)

_Another one from SUSE_

### CODE REVIEW

File: lib/rpmrc.c [hunk 1]
```diff
@@ -1103,6 +1111,12 @@ static void defaultMachine(rpmrcCtx ctx, const char ** arch, const char ** os)
 #		endif
 #	endif
 
+#if defined(__linux__)
+	/* in linux, lets rename parisc to hppa */
+	if (rstreq(un.machine, "parisc"))
+	    strcpy(un.machine, "hppa");
```
Comment (Conan-Kudo) [inline lib/rpmrc.c]: This feels like an unrelated change. As far as I know the Crusoe has nothing to do with `hppa`...

Comment (simotek) [inline lib/rpmrc.c]: Done

**Comment by ffesti** [comment]
Thanks! Pushed.
